cache/cache.go

```diff
@@ -82,6 +82,7 @@ type BlobCache interface {
 type Reader interface {
 	io.ReaderAt
 	Close() error
+	GetReaderAt() io.ReaderAt
```

Comment:
When FUSE passthrough is enabled,  we should always set `directoryCache.direct` to `true` so that we can ensure that `*directoryCache.Get` always return *os.File (not a buffer).